projects/experimental/directives/button-group-card/button-group-card.style.less

```diff
@@ -0,0 +1,52 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroupCard] {
+    display: flex;
+    min-width: 21.4375rem;
+    border-radius: 1.5rem;
+
+    &[data-surface='dark'] > [tuiButton][tuiAppearance] {
+        color: #fff;
+
+        &:hover {
+            background: rgba(256, 256, 256, 0.13);
+        }
+    }
+
+    > [tuiButton][tuiButtonVertical] {
+        flex-grow: 1;
+        flex-basis: 0;
+        border-radius: 0;
+        padding: 1.125rem 0.25rem;
```

Comment:
Why do we need to override these last 3 styles?

Reply:
with

```css
padding: 1.125rem 0.25rem;
min-width: 6.625rem;
gap: 0.5rem;
```

<img width="374" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/f74a1df4-d80b-43de-997e-94b1bcd66b1f">

<img width="683" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/693316b5-6c61-4487-80f3-a5ea132bc0d8">


without 

<img width="336" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/2b26ca36-5a7d-474d-a051-e7974eecc944">